doc/release-notes.md

```diff
@@ -466,14 +150,16 @@ Thanks to everyone who directly contributed to this release:
 - Pieter Wuille
 - poole\_party
 - practicalswift
-- R E Broadley
-- randy-waterhouse
-- Richard Kiss
-- Robert McLaughlin
-- rodasmith
+- Primo
+- Reiner Herrmann
 - Ross Nicoll
-- Russell Yanofsky
-- Shigeya Suzuki
+- Shibe
+- Sjors Provoost
+- Suhas Daftuar
+- Thomas Snider
+- tnaka
+- Tjps
+- Vertian
 - Wladimir J. van der Laan
```

Comment:
I only have the following contributors when iterating the commits and removing reversals:

```
- Anthony Chen 
- BT 
- Bertrand Jacquin 
- Daniel Edgecumbe 
- Dennis Field 
- Ikko Ashimine 
- John-Gee 
- Jonathan 
- Kent 
- Luis-Johannes Schubert 
- Marco 
- Max K 
- Patrick Lodder 
- Primo 
- Reiner Herrmann 
- Ross Nicoll 
- Shibe 
- Vertian 
- marcuswin 
```